feature-staking-impl/src/main/java/io/novafoundation/nova/feature_staking_impl/domain/nominationPools/claimRewards/validations/ValidationFailureUi.kt

```diff
@@ -15,5 +15,8 @@ fun nominationPoolsClaimRewardsValidationFailure(
 
         NominationPoolsClaimRewardsValidationFailure.NonProfitableClaim -> resourceManager.getString(R.string.common_confirmation_title) to
             resourceManager.getString(R.string.staking_warning_tiny_payout)
+
+        is NominationPoolsClaimRewardsValidationFailure.ToStayAboveED -> resourceManager.getString(R.string.common_not_enough_funds_title) to
+            resourceManager.getString(R.string.wallet_send_insufficient_balance_commission, failure.asset.symbol)
```

Comment:
Can we make a shared handler to not to duplicate title and message in each case (unstake, redeem, claim)?

Reply:
As far as I know we do not adhere to this practice for validation since we have different Failure classes for each validation

Reply:
We can actually move this to a separate function by using a separate interface for validation error. We already use it in separate placed, for example NotEnoughToPayFees validation